### PR TITLE
utils: add glibc version checking for uadk_memcpy

### PR DESCRIPTION
**Pull request by yekai123123**
No assembly language is required as glibc version >= 2.32. This
is because the later glibc version has updated this optimization.

Signed-off-by: Kai Ye <yekai13@huawei.com>

### CODE REVIEW

**Review by gaozhangfei — COMMENTED** [review]
can we remove memcpy
               priv->req.in_bytes = DIGEST_BLOCK_SIZE;
                priv->req.in = src;

next time
                priv->req.in_bytes = len - DIGEST_BLOCK_SIZE;
                priv->req.in = src+ DIGEST_BLOCK_SIZE;

**Comment by docularxu** [comment]
This doesn't fix the problem of building failure on x86 platforms or any other non-NEON capable platforms.

File: src/uadk_utils.c [hunk 1]
```diff
@@ -56,7 +59,9 @@ static void *memcpy_large(void *dstpp, const void *srcpp, size_t len)
 
 void *uadk_memcpy(void *dstpp, const void *srcpp, size_t len)
 {
-	if (len >= UADK_MEM_IMPROVE_THRESHOLD)
+	/* No assembly language is required as glibc version >= 2.32 */
+	if (((__GLIBC__ == GLIBC_MAIN && __GLIBC_MINOR__ < GLIBC_MINOR) ||
```
Comment (docularxu) [inline src/uadk_utils.c:63]: Don't make this a runtime comparison. Make it as a compilation time preprocessing. Refer this:
https://chromium.googlesource.com/v8/v8/+/refs/heads/main/include/v8config.h#36

Comment (yekai123123) [reply src/uadk_utils.c]: done

**Pull request closed without merging**